web/themes/custom/bart/src/scss/footer.scss

```diff
@@ -0,0 +1,225 @@
+$link: #f5b000;
+
+.featured-bottom {
+  background-color: white;
+  border: 0;
+
+  //Design h2 and remove double title and border
+  h2 {
+    text-align: left;
+    font-size: 1.75rem;
+    font-weight: 800;
+    padding: 0 0 10px 0;
+  }
+  .node__title, .node__meta {
+    display: none;
+  }
+  a {
+    border-bottom: none;
+  }
+  // Editor picks and must read blocks
+  .region-featured-bottom-first, .region-featured-bottom-second {
+
+  .block-field-blocknodearticlestitle {
+    margin: 0;
+    border: 0;
+    padding: 0;
+  }
+
+  .content {
+    margin: 0;
+  }
+
+    a {
+      font-weight: 600;
+      font-size: 1rem;
+    }
+  }
+  // Hot topics
+  .region-featured-bottom-third {
+    td {
+      font-size: 1rem;
+      font-weight: 600;
+    }
+    tr {
+      background: none;
+    }
+  }
+
+}
+
+// Config for all footer regions
+.site-footer {
+  background: white;
+  padding: 0;
+
+
+
+  .layout-container {
+    margin: 0;
+    padding: 0;
+    min-width: 100%;
+    // theming footer five region
+    .region-footer-fifth {
+
+      // theming third column with soc ic
+      .col-3 .field--type-entity-reference {
+        display: flex;
+
+        .image-style-thumbnail {
+          padding: 0 20px 0 0;
+        }
+      }
+
+      // theming second column and titles
+      .layout--onecol {
+        .field--name-field-header {
+          font-size: 1.75rem;
+          color: black;
+          font-weight: 800;
+        }
+
+        p {
+          font-size: 0.875rem;
+          color: grey;
+          padding: 0 20px 0 0;
+        }
+
+        a {
+          color: $link;
+          border-bottom: none;
+        }
+
+      }
+
+      // theming logo LifeNews
+      .layout--twocol-section--25-75 {
+        width: 100%;
+        padding: 2.8rem 0;
+        margin: 0 0 0 15rem;
+
+        .content {
+          margin-top: 0;
+          padding: 0;
+
+        }
+
+        .block {
+          margin: 0;
+        }
+
+        .image-style-thumbnail {
+          float: right;
+          padding: 0 0.625rem 0 0;
+        }
+
+        p {
+          margin: 0;
+        }
+
+        .field--type-text {
+          font-weight: 700;
+          font-size: 1.875rem;
+          color: black;
+        }
+
+        .field--type-text-with-summary {
+          font-size: 0.625rem;
+          font-weight: 300;
+          color: black;
+        }
+
+        .layout__region--second {
+          padding: 15px 0;
+        }
+
+        .field--type-entity-reference {
+          padding: 20px 0;
+        }
+      }
+
+      // delete border customise block with photos
```

Comment:
Read about comments standards:
1) Should start with a uppercase letter
2) Should contain dot at the end.